tests/buildtest.cmd

```diff
@@ -359,6 +362,7 @@ echo     666: Build all tests with priority 0, 1 ... 666
 echo sequential: force a non-parallel build ^(default is to build in parallel
 echo     using all processors^).
 echo IlasmRoundTrip: enables ilasm round trip build and run of the tests before executing them.
+echo updateinvalidpackages: enables updating package versions in all test project.json files
```

Comment:
I tried this out locally through the powershell script, and it looks like it does some unnecessary building of native components and a restore for CoreDisTools. It would be good to do as little as possible to reduce dependencies needed for auto-upgrade build machines and try to remove any dependency on a successful test build for auto-upgrade to work. (There's no special reporting for the auto-upgrade system, so "catching" a test build failure there isn't very useful.)

It did work on my machine, though! I tried each element and they were changed correctly, and for CoreFX, the project.jsons were upgraded.